ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -130,23 +199,14 @@ export class AdminWorkspace extends React.Component<{}, State> {
           Search
         </Button>
       </FlexRow>
-      {
-        workspace && <FlexColumn>
-          <h3>Workspace Admin Actions</h3>
-          <FlexRow style={{justifyContent: 'space-between'}}>
-            <Button>Shut down all VMs</Button>
-            <Button>Disable workspace</Button>
-            <Button>Disable all collaborators</Button>
-            <Button>Exclude from public directory</Button>
-            <Button>Log administrative comment</Button>
-            <Button>Publish workspace</Button>
-          </FlexRow>
-        </FlexColumn>
-      }
-      {workspace && resources.workspaceObjects && resources.cloudStorage && collaborators &&
-        <FlexRow>
-          <FlexColumn style={styles.wideWithMargin}>
-            <h3>Basic Information</h3>
+
+      {loadingData && <SpinnerOverlay />}
+
+      <h2>Workspace</h2>
```

Comment:
This header is going to show up even if we haven't searched a project. When I wrote this page I deliberately made sure it wouldn't. Not the biggest deal for an internal page though.